contracts/AccessNFT.sol

```diff
@@ -294,6 +313,15 @@ contract AccessNFT is ERC1155PresetMinterPauserSupplyHolder, ERC2771Context, IER
         emit RestrictedTransferUpdated(_restrictedTransfer);
     }
 
+    /// @dev Lets a module admin set the URI for contract-level metadata.
+    function setOwner(address _newOwner) external onlyModuleAdmin {
+        require(hasRole(DEFAULT_ADMIN_ROLE, _newOwner), "new owner not module admin.");
+        address _prevOwner = _owner;
```

Comment:
one pattern i learned recently is that you could just emit first, you don't have to copy `_prevOwner`

for example:
```
require(...)
emit NewOwner(_owner, _newOwner);
_owner = _newOwner
```